m/speciesAbundance/speciesAbundance.R

```diff
@@ -98,7 +98,8 @@ abundanceSim <- function(sim) {
 }
 
 ## This is not an event, but a function that we define separately 
-## and that contains our "simulation model"
+## and that contains our "simulation model". 
+## It is possible to specify it as a separate script (which is worth it if the function is too big, for example).
```

Comment:
instead:
It is possible to specify it as a separate script (which is worth doing if the function is too big, for example).